app/assets/javascripts/webchat.js

```diff
@@ -1,8 +1,6 @@
 //= require ./webchat/library.js
 
-var $ = window.$
-
-$(document).ready(function () {
+document.addEventListener('DOMContentLoaded', function () {
```

Comment:
Recently had to do this exact change in another script and Kevin pointed out it would work just fine by removing the `document.ready` entirely without replacing it. Can you test whether this would work?

Reply:
You might also be able to to do the following:

```js
(function () {
.... your code
})()
```

Reply:
I think a self executing function in this context would be the same as having the code execute at this point without a document.ready.